lighthouse-core/audits/user-timings.js

```diff
@@ -125,13 +125,22 @@ class UserTimings extends Audit {
     };
   }
 
+  // We'll remove mark/measures entered by third parties not of interest to the user
+  static get blacklistedPrefixes() {
+    return ['goog_'];
+  }
+
+  static excludeBlacklisted(timing) {
```

Comment:
jsdoc would be good (nit: and maybe comment above should actually go on this function)